python/src/uagents/context.py

```diff
@@ -364,7 +365,7 @@ async def experimental_broadcast(
             ]
         )
         self.logger.debug(f"Sent {len(futures)} messages")
-        return futures
+        return list(futures)
```

Comment:
Likewise here @Archento I think that `asyncio.gather` returns a `tuple()` - can you and @jrriehl give this a double check

Reply:
I just checked the example with 0..2 responses and all were lists.
According to the [docs](https://docs.python.org/3/library/asyncio-task.html#running-tasks-concurrently) all successful awaitables will be aggregated to a list and as we always return with a `MsgStatus` this should therefore also always be a list meaning that we shouldn't need the explicit casting although it doesn't hurt either.

Reply:
Reverting this change due to @Archento's findings.